pkg/components/components.go

```diff
@@ -490,6 +490,11 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 			Resources: stringListToSlice("namespaces"),
 			Verbs:     stringListToSlice("get", "list", "watch", "patch", "update"),
 		},
+		{
+			APIGroups: emptyAPIGroup,
+			Resources: stringListToSlice("resourcequotas"),
+			Verbs:     stringListToSlice("get", "list", "watch"),
+		},
```

Comment:
nit: The rback section for pods shares the same setting. it is possible to merge them, to get shorter CSV.

Reply:
:+1: